profiles/repositories.py

```diff
@@ -66,6 +70,14 @@ def get(self, orcid: str) -> OrcidToken:
     def clear(self) -> None:
         self.db.session.query(OrcidToken).delete()
 
+    def remove(self, orcid: str) -> None:
+        try:
+            orcid_token = self.db.session.query(OrcidToken).filter_by(orcid=orcid).one()
+            self.db.session.delete(orcid_token)
+        except NoResultFound as exception:
+            raise OrcidTokenNotFound('ORCID token for the ORCID {} not found'.format(orcid)) \
```

Comment:
Not sure there's a need to throw an exception here? If you try and remove something that doesn't exist, 🤷‍♂️ .

Reply:
OK change to `pass` or log the exception?